cyfra-rtrp/src/main/scala/io/computenode/cyfra/rtrp/surface/SurfaceIntegrationExample.scala

```diff
@@ -0,0 +1,172 @@
+package io.computenode.cyfra.rtrp.surface
+
+import io.computenode.cyfra.rtrp.window.WindowManager
+import io.computenode.cyfra.rtrp.window.WindowManager.*
+import io.computenode.cyfra.rtrp.window.core.WindowConfig
+import io.computenode.cyfra.rtrp.window.core.WindowPosition
+import io.computenode.cyfra.rtrp.surface.core.*
+import io.computenode.cyfra.vulkan.VulkanContext
+import scala.util.*
+
+// Complete example demonstrating the integrated window + surface system.
+object SurfaceIntegrationExample:
+
+  def main(args: Array[String]): Unit =
+    println("=== Cyfra Surface Integration Example ===")
```

Comment:
Try using the logger instead of printlns - import `io.computenode.cyfra.utility.Logger.logger` and then you can do logger.debug(...), logger.info(...), logger.warn(..)

Reply:
It is better because it is configurable on the side of the library user - they can write a configuration of where they want the logs to go and how they want them to be formatted. Usually in production apps you will use loggers instead  of println.